MultiLangProgGuide/MultiLangProgGuide.rst

```diff
@@ -2717,23 +2771,133 @@ Go Comprehensive API
+for more information.
 
-GetSubBufAlloc(n)
-GetSubBufUsed(n)
+:code:`YDBForkNCore()` has no parameters and returns no return code.
+
+------------
+Go YDBFree()
+------------
+
+.. code-block:: go
+
+    YDBFree(ptr unsafe.pointer)
+
+The function wraps `ydb_free()`_ to release memory previously
+allocated using :code:`YDBMalloc()`. As passing a :code:`ptr` not
+previously allocated using :code:`YDBMalloc()` will result in
+unpredictable behavior, application code should be written with an
+appropriate level of diligence when calling :code:`YDBFree()`.
+
+------------
+Go YDBInit()
+------------
+
+.. code-block:: go
+
+    YDBInit() int
+
+The function wraps `ydb_init()`_ to initialize the YottaDB runtime
+system. This call is normally not required as YottaDB initializes
+itself on its first call, the exception being when an application
+wishes to set its own signal handlers (see
+`Signals`_). :code:`YDBInit()` returns :code:`C.YDB_OK` or an `error
+return code`_.
+
+--------------
+Go YDBMalloc()
+--------------
+
+.. code-block:: go
+
+    YDBMalloc(size uint64) unsafe.pointer
+
+The function wraps `ydb_malloc()`_ to allocate :code:`size` bytes of
+storage managed by YottaDB. Use of :code:`YDBMalloc()` to allocate
+storage provides debugging tools. Using any function other than
+:code:`YDBFree()` to release storage allocated with
+:code:`YDBMalloc()` has unpredictable results.
+
+As the definition and implementation of Go protect against dangling
+pointers in pure Go code, Go application code may not be designed and
+coded with the same level of defensiveness against dangling pointers
+that C applications are. Please take additional care in designing and
+coding your application to ensure the correct use of application
+storage allocated using :code:`YDBMalloc()`.
 
 --------------------------
-Go BufferT Utility Methods
+Go YDBStdoutStdErrAdjust()
 --------------------------
 
---------------------
-Go Utility Functions
---------------------
+.. code-block:: go
+
+    YDBStdoutStderrAdjust() int
+
+Refer to the description of `ydb_stdout_stderr_adjust()`_, which this
+function wraps. The return code is :code:`C.YDB_OK`.
+
+Applications that do not mix M and non-M code do not need this
+function.
+
+-------------------
+Go YDBTimerCancel()
+-------------------
+
+.. code-block:: go
+
+    YDBTimerCancel(timerid uintptr)
+
+The function wraps `ydb_timer_cancel()`_ to cancel a timer previously
+established using :code:`YDBTimerStart()`. :code:`timerid` is the id
+of the timer. The function returns no error code.
+
+------------------
+Go YDBTimerStart()
+------------------
+
+.. code-block:: go
+
+    YDBTimerStart(timerid uintptr,
+        limitNsec uint64,
+	handler unsafe.pointer,
+	handlerDataLen uint,
+	handlerData unsafe.pointer) int
+
+The function wraps `ydb_timer_start()`_ to start a timer. Unless
+canceled, after the timer expires, YottaDB invokes the handler
+function referenced by 
```

Comment:
Incomplete sentence.

Reply:
That's because you were reviewing functions before I finished drafting the functions content. I had said to stop after Methods!